pywwt/quantity_trait.py

```diff
@@ -0,0 +1,12 @@
+from traitlets import TraitType
+from astropy import units as u
+
+class AstropyQuantity(TraitType):
+
+    default = 0 * u.deg
```

Comment:
I don't think the default should have any specific units. Use ``0 * u.one`` to mean dimensionless (but still be an Astropy quantity)